orbit/cmd/desktop/desktop.go

```diff
@@ -271,9 +296,33 @@ func main() {
 				}
 				myDeviceItem.Enable()
 
-				if sum.Notifications.NeedsMDMMigration {
+				if runtime.GOOS == "darwin" &&
+					(sum.Notifications.NeedsMDMMigration || sum.Notifications.RenewEnrollmentProfile) &&
+					mdmMigrator.CanRun() {
+
+					// if the device is unmanaged or we're
+					// in force mode and the device needs
+					// migration, enable aggresive mode.
+					aggresive := sum.Notifications.RenewEnrollmentProfile ||
```

Comment:
Typo: `aggressive` (not providing a diff suggestion because it needs to be renamed throughout this scope). Also, feel free to ignore but when there's a mix of ANDs and ORs I like to use parens to make the condition more obvious to the future reader, not having to second guess operator precedence.